contracts/BSCValidatorSet.sol

```diff
@@ -371,16 +376,16 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     }
   }
 
-  function getMiningValidators() public view returns(address[] memory) {
+  function getMiningValidators() external view override returns(address[] memory, bytes[] memory) {
     uint256 _maxNumOfWorkingCandidates = maxNumOfWorkingCandidates;
     uint256 _numOfCabinets = numOfCabinets;
     if (_numOfCabinets == 0 ){
       _numOfCabinets = INIT_NUM_OF_CABINETS;
     }
 
-    address[] memory validators = getValidators();
+    (address[] memory validators, bytes[] memory voteAddrs) = getValidators();
```

Comment:
We are changing the interface of `getValidators`,  some external service is using it, lets come up workaround without changing it.

Reply:
Fixed